docs/help/Changing-Plugin-Types-In-Starfield.html

```diff
@@ -12,17 +14,16 @@
 <ul>
   <li><code>Full Master</code> are standard <code>ESMs</code>.</li>
   <li><code>Medium Master</code> use the load order slot <code>FD</code> and allow 65.535 records each in up to 256 plugins.</li>
-  <li><code>Small Master</code> use the load order slot <code>FE</code> and allow 4.095 records each in up to 4096 plugins. These plugins are what was previously known as <code>Light</code> masters.</li>
+  <li><code>Small Master</code> use the load order slot <code>FE</code> and allow 4.095 records each in up to 4.096 plugins. These plugins are what was previously known as <code>Light</code> masters.</li>
```

Comment:
For this & the previous line, we should probably avoid using period/comma separators for numbers to avoid confusion as much as possible.

Reply:
Sure, we can leave them out.

Reply:
Done in https://github.com/loot/loot.github.io/commit/b133cc129051af155162cf3d4065f4231b2f067d